src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -306,7 +306,7 @@ public static void makeGenotypeCall(final int ploidy,
                                         final GenotypeAssignmentMethod assignmentMethod,
                                         final double[] genotypeLikelihoods,
                                         final List<Allele> allelesToUse,
-                                        final List<Allele> originalGT,
+                                        final Genotype originalGT,
```

Comment:
I had to change this method signature so that we could use the GQ info to distinguish hom-ref genotypes that we want to no-call from hom-refs that stay hom-refs because with reblocking there are no PLs to use.

Reply:
add to method doc